Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "b64-ct"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Need to bump the version again. Last PR made it `0.1.2`.
Will push a commit for that.